documentation/rfc/journeys_ridesharing.json

```diff
@@ -0,0 +1,556 @@
+{
+    "tickets": [
+        {
+            "comment": "Ridesharing price 1",
+            "cost": {
+                "value": "12.40",
+                "currency": "EUR"
+            },
+            "found": false,
+            "id": "ridesharing_price_1",
+            "links": [
+                {
+                    "id": "section_6_1_1",
+                    "internal": true,
+                    "rel": "sections",
+                    "templated": false,
+                    "type": "section"
+                }
+            ],
+            "name": "ridesharing_price_01"
+        }
+    ],
+    "links": [],
+    "journeys": [
+        {
+            "status": "",
+            "distances": {
+                "ridesharing": 138097,
+                "car": 0,
+                "walking": 0,
+                "bike": 0
+            },
+            "links": [
+                {
+                    "href": "https://api.navitia.io/v1/coverage/rennes/journeys?allowed_id%5B%5D=stop_point%3AOCE%3ASP%3ATGV-87318964&allowed_id%5B%5D=stop_point%3AOCE%3ASP%3ATGV-87286005&from=admin%3Afr%3A12145&last_section_mode%5B%5D=car&max_duration_to_pt=86400&is_journey_schedules=True&direct_path=none&to=admin%3Afr%3A59350&min_nb_journeys=5&car_speed=20&min_nb_transfers=0&debug=false&first_section_mode%5B%5D=car",
+                    "type": "journeys",
+                    "rel": "same_journey_schedules",
+                    "templated": false
+                }
+            ],
+            "tags": [
+                "ridesharing",
+                "ecologic"
+            ],
+            "nb_transfers": 0,
+            "durations": {
+                "ridesharing": 9763,
+                "car": 0,
+                "walking": 0,
+                "total": 27343,
+                "bike": 0
+            },
+            "arrival_date_time": "20180110T194400",
+            "calendars": [],
+            "departure_date_time": "20180110T120817",
+            "requested_date_time": "20180110T113634",
+            "fare": {
```

Comment:
so can we use this section to put ridesharing fare?

Reply:
not this one, maybe the embedded one.